docs/about/release-notes.rst

```diff
@@ -32,6 +32,7 @@ Breaking changes
   It is also the only interactive backend that works in JupyterLab.
 * Removed support for ``event_list`` ``dtype``, use binned data instead.
 * Removed support for "realigned" data. This is replaced by the more flexible and generic support for "binned" data.
+* Experimental support for saving and loading scipp data structures to HDF5.
```

Comment:
Why experimental? unless you are thinking that HDF5 support for scipp objects might be dropped.

Reply:
Because we haven't really tested it much in practice?

Reply:
I think the term is OK. My hesitation was more along the lines that we already encode the version information, so in principle we can make future changes without breaking backward capability (if we want).

Basically leave as is.